irohad/ametsuchi/mutable_storage.hpp

```diff
@@ -71,7 +74,7 @@ namespace iroha {
        * @return True if block was successfully applied, false otherwise.
        */
       virtual bool apply(const shared_model::interface::Block &block,
-                         MutableStoragePredicateType function) = 0;
+                         MutableStoragePredicate<WsvQuery> function) = 0;
```

Comment:
It should be possible to remove the predicate from this method
https://github.com/hyperledger/iroha/blob/edf8ee73af2dcbeefa1471dde16de5b08a4fd362/irohad/synchronizer/impl/synchronizer_impl.cpp#L29-L31